tests/functions/test_memory.py

```diff
@@ -499,7 +499,7 @@ def test_DictionaryMemory_add_and_delete_from_memory(self):
                            [[ 1,  2,  3],[100,101,102,103]]]
         for m,e in zip(em.memory,expected_memory):
             for i,j in zip(m,e):
-                assert np.allclose(i,j)
```

Comment:
Can the zip iterations be replaced by a single call to `np.testing.assert_allclose`?

Reply:
No, ragged array